config/spryk/spryks/Spryker/Zed/Persistence/Propel/Schema/AddZedPersistencePropelSchemaProperty.yml

```diff
@@ -11,18 +10,25 @@ arguments:
         inherit: true
         default: Spryker
 
-    name:
-        inherit: true
-
     propertyName: ~
 
     propertyType: ~
 
-    singular:
-        default: false
+    required: ~
+
+    autoIncrement: ~
+
+    primaryKey: ~
+
+    defaultValue: ~
 
     targetPath:
-        value: "src/{{ organization }}/Zed/{{ module }}/Persistence/Propel/Schema/{{ module | underscored }}.schema.xml"
+        value: "src/{{ organization }}/Zed/{{ module }}/Persistence/Propel/Schema/spy_{{ module | underscored }}.schema.xml"
 
 preSpryks:
-    - AddZedPersistencePropelSchema
+    - AddZedPersistencePropelSchemaTable:
+          arguments:
+              tableName:
+                  value: "spy_{{ module | underscored }}"
+              targetModule:
```

Comment:
Can't you re-use `module`?

Reply:
If I remove targetModule from here and replace it with module or something else, everything breaks. Here now the module is passed to the value, I guess I didn't get your point completely

Reply:
If it only works with `targetModule` I'm fine.